docs/docusaurus/README.md

```diff
@@ -78,17 +78,18 @@ To add a new version, follow these steps:
 
 1. It may help to start with a fresh virtualenv and clone of gx.
 2. Check out the version from the tag, e.g. `git checkout 0.15.50`
-3. Make sure dev dependencies are installed `pip install -c constraints-dev.txt -e ".[test]"`
-4. Install API docs dependencies `pip install -r docs/sphinx_api_docs_source/requirements-dev-api-docs.txt`
-5. Build API docs `invoke api-docs` from the repo root.
-6. Run `yarn install` from `docs/docusaurus/`.
-7. Temporarily change onBrokenLinks: 'throw' to onBrokenLinks: 'warn' in `docusaurus.config.js` to allow the build to complete even if there are broken links.
-8. Run `yarn build` from `docs/docusaurus/`.
-9. Create the version e.g. `yarn docusaurus docs:version 0.15.50` from `docs/docusaurus/`.
-10. Pull down the version file (see `docs/build_docs` for the file, currently https://superconductive-public.s3.us-east-2.amazonaws.com/oss_docs_versions.zip)
-11. Unzip and add your newly created versioned docs via the following:
-12. Copy the version you built in step 4 from inside `versioned_docs` in your repo to the `versioned_docs` from the unzipped version file.
-13. Copy the version you built in step 4 from inside `versioned_sidebars` in your repo to the `versioned_sidebars` from the unzipped version file.
-14. Add your version number to `versions.json` in the unzipped version file.
-15. Zip up `versioned_docs`, `versioned_sidebars` and `versions.json` and upload to the s3 bucket (see `docs/build_docs` for the bucket name). Make sure `versioned_docs`, `versioned_sidebars` and `versions.json` are at the top level of the zip file (not nested in a folder).
-16. Once the docs are built again, this zip file will be used for the prior versions.
+3. Modify `docs/docusaurus/docusaurus.config.js` and `docs/docusaurus/docs/components/_data.jsx` to the new version (the patch version may be off by 1 e.g. 0.17.22 instead of 0.17.23 since we update this in the release PR which comes after the release tag).
+4. Make sure dev dependencies are installed `pip install -c constraints-dev.txt -e ".[test]"` and `pip install pyspark`
+5. Install API docs dependencies `pip install -r docs/sphinx_api_docs_source/requirements-dev-api-docs.txt`
+6. Build API docs `invoke api-docs` from the repo root.
```

Comment:
Out of scope here, but AFAIK markdown auto-numbers numbered lists regardless of the numbers we put. So we could just use `1.` for each, and it would render correctly.https://www.markdownguide.org/basic-syntax/#ordered-lists. Just FYI if we want to cut down on PR noise in the future!

Reply:
Good call - might as well do it now since the diff is already noisy.